CHANGELOG/v1.4.9.md

```diff
@@ -0,0 +1,32 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.23.x -> v1.27.x
+- Workload Cluster: v1.21.x -> v1.27.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.4.8
+## :chart_with_upwards_trend: Overview
+- 14 new commits merged
+- 4 bugs fixed 🐛
+
+## :bug: Bug Fixes
+- CAPD: Fix DockerMachine panic (#9690)
```

Comment:
Sorry one PR to add: https://github.com/kubernetes-sigs/cluster-api/pull/9809

(cherry-pick didn't make it before because of merge conflicts)